common_settings.bzl

```diff
@@ -199,7 +199,12 @@ WINDOWS_COMMON_STATIC_LIBS_COPTS = [
                         "/wd4702",
                         "/wd4267",
                         "/wd4996",
+                        "/wd6240", 
```

Comment:
why new flags are added?

Reply:
There are new warnings in fmt treated as errors. Spdlog compilation requires  "/utf-8" flag set for windows compilation.